Source/Csla.Xaml.Maui/Csla.Xaml.Maui.csproj

```diff
@@ -1,6 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\Directory.Package.props" />
```

Comment:
Is that necessary? Something MAUI specific?

Reply:
The project type? That is MAUI specific.